docs/Cube.js-Backend/Security.md

```diff
@@ -5,54 +5,52 @@ category: Cube.js Backend
 menuOrder: 4
 ---
 
-Cube.js uses [JSON Web Tokens (JWT)](https://jwt.io/) which passed in `Authorization` header for requests' authorization and also for passing
-additional user context, which could be used in the [USER_CONTEXT](cube#context-variables-user-context) object in the Data
-Schema.
-`Authorization` header is parsed and set to [authInfo](@cubejs-backend-server-core#authinfo) variable which is also can be used for [Multitenancy](multitenancy-setup).
+Cube.js uses [JSON Web Tokens (JWT)](https://jwt.io/) which should be passed in the `Authorization` header to authenticate requests. JWTs can also be used for passing
+additional information about the user, which can be accessed in the [USER_CONTEXT](cube#context-variables-user-context) object in the Data
+Schema. 
 
-Cube.js tokens are designed to work in micro services environment.
-Typical use case would be:
+The
+`Authorization` header is parsed and the JWT's contents set to the [authInfo](@cubejs-backend-server-core#authinfo) variable which can be used to support [Multitenancy](multitenancy-setup).
 
-1. There's web server that serves HTML with JS client code that calls cube.js.
-2. Web server should generate expirable cube.js tokens and incorporate them as part of HTML or send it over XHR request in exchange of session cookie or other security credentials.
-3. JS Client code uses token to call cube.js server API.
+Cube.js tokens are designed to work well in microservice-based environments. Typical use case would be:
 
-If you are using [REST API](rest-api) you need pass API Token via the Authorization Header.
-Cube.js Javascript client accepts auth token as a first argument to [cubejs(authToken, options) function](@cubejs-client-core#cubejs).
+1. A web server serves an HTML page containing the cube.js client, which needs to communicate securely with the cube.js back-end.
+2. The web server should generate an expirable cube.js token to achieve this. The server could include the token in the HTML it serves or provide the token to the front-end via an XHR request, which stores it in local storage or a cookie.
+3. The JS client is initiallised using this token, and includes it in calls to the cube.js server API.
```

Comment:
a typo here `initiallised`. And I guess we should prefer American spelling for consistency.